adoc/retail_chap_install.adoc

```diff
@@ -184,9 +184,13 @@ For instructions to install Salt minion see xref:quickstart3_chap_suma_keys_and_
 
 [WARNING]
 ====
+The build host must be a Salt minion.
 Do not install the build host as a traditionally managed client.
 ====
 
+When you are installing the build host, check that you are using the same fully qualified domain name (FQDN) on both the external and internal branch networks.
+If the FQDN does not match on both networks, the build host will not be recognized as a proxy server.
+
```

Comment:
This is not about build host, but branch server. Can you move it down to the `retail.sect.install.branch` section? And adapt the text (emphasis on changes):

> When you are installing the **Branch server with dedicated internal network**, check that you are using the same fully qualified domain name (FQDN) on both the external and internal branch networks.
If the FQDN does not match on both networks, the **Branch server** will not be recognized as a proxy server.